applications/StructuralMechanicsApplication/custom_utilities/structural_mechanics_element_utilities.cpp

```diff
@@ -58,30 +63,51 @@ double GetRayleighAlpha(
     // over the globally defined one (through ProcessInfo)
     if (rProperites.Has(RAYLEIGH_ALPHA)) {
         return rProperites[RAYLEIGH_ALPHA];
-    }
-    else if (rCurrentProcessInfo.Has(RAYLEIGH_ALPHA)) {
+    } else if (rCurrentProcessInfo.Has(RAYLEIGH_ALPHA)) {
         return rCurrentProcessInfo[RAYLEIGH_ALPHA];
     }
 
     return 0.0;
 }
 
+/***********************************************************************************/
+/***********************************************************************************/
+
 double GetRayleighBeta(
     const Properties& rProperites,
     const ProcessInfo& rCurrentProcessInfo)
 {
-    // giving the locally defined setting (through Properties) priority
+    // Giving the locally defined setting (through Properties) priority
     // over the globally defined one (through ProcessInfo)
     if (rProperites.Has(RAYLEIGH_BETA)) {
         return rProperites[RAYLEIGH_BETA];
-    }
-    else if (rCurrentProcessInfo.Has(RAYLEIGH_BETA)) {
+    } else if (rCurrentProcessInfo.Has(RAYLEIGH_BETA)) {
         return rCurrentProcessInfo[RAYLEIGH_BETA];
     }
 
     return 0.0;
 }
 
+/***********************************************************************************/
+/***********************************************************************************/
+
+double GetDensity(const Element& rElement)
+{
+    // Getting the properties of the element
+    const auto& r_prop = rElement.GetProperties();
+
+    // Getting the density of the element
+    const double density = r_prop[DENSITY];
+
+    // Getting the mass factor
+    const double mass_factor = rElement.Has(MASS_FACTOR) ? rElement.GetValue(MASS_FACTOR) : r_prop.Has(MASS_FACTOR) ? r_prop.GetValue(MASS_FACTOR) : 1.0;
```

Comment:
1. can this be an elemental-data?

2. can you please follow the style of this file, which is more readable IMO?
~~~c


if (rElement.Has(MASS_FACTOR)) {
    return rElement.GetValue(MASS_FACTOR) * density;
} else if (r_prop.Has(MASS_FACTOR)) {
    return r_prop.GetValue(MASS_FACTOR) * density;
} else {
    return density;
}
~~~
3. Please add tests in the same way did such that the correct defaults are tested

Reply:
Ok